ansi/width.go

```diff
@@ -37,16 +31,21 @@ func Strip(s string) string {
 			pstate = parser.GroundState
 			ri = 0
 			rw = 0
-		case action == parser.PrintAction:
-			// This action happens when we transition to the Utf8State.
-			if w := utf8ByteLen(s[i]); w > 1 {
-				rw = w
+			continue
+		}
+
+		state, action := parser.Table.Transition(pstate, s[i])
+		switch action {
+		case parser.CollectAction:
+			if state == parser.Utf8State {
+				// This action happens when we transition to the Utf8State.
+				rw = utf8ByteLen(s[i])
 				buf.WriteByte(s[i])
 				ri++
-				break
 			}
+		case parser.PrintAction:
 			fallthrough
```

Comment:
A fall through for this seems over kill

```suggestion
		case parser.PrintAction, parser.ExecuteAction:
```